mac

```diff
@@ -151,11 +150,27 @@ cask "docker"
 brew "postgres"
 brew "redis"
 
-# Basics
-cask "google-chrome"
-cask "slack"
 EOF
 
+# Basics Apps
+
+if [ ! -d "/Applications/Google Chrome.app" ]; then
+  fancy_echo "Installing Google Chrome.."
+  brew cask install google-chrome
+  fancy_echo "Google Chrome installed."
+fi
+
+if [ ! -d "/Applications/Slack.app" ]; then
+  fancy_echo "Installing Slack.."
+  brew cask install slack
+  fancy_echo "Slack installed."
+fi
```

Comment:
Does it really requires to check if the app is there or not? I

Reply:
I installed downloaded Google Chrome first before running the script, and it just shows error and cannot continue the script. So yeah it causes problem if you have Chrome or Slack manually installed beforehand.